### PR TITLE
Added Leave Meeting Button to Handwired folder

### PR DESCRIPTION
**Pull request by aerialviews007**
I 3D printed and assembled the Leave Meeting button created by 9gel.  https://github.com/9gel/leave_meeting_button

Rather than use his software, I installed QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/leave_meeting/config.h [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 Keebio
```
Comment (tzarc) [inline keyboards/handwired/leave_meeting/config.h:2]: Wrong author.

File: keyboards/handwired/leave_meeting/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 Keebio
```
Comment (tzarc) [inline keyboards/handwired/leave_meeting/keymaps/default/config.h:1]: Wrong author.

File: keyboards/handwired/leave_meeting/config.h [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    You
```
Comment (tzarc) [inline keyboards/handwired/leave_meeting/config.h:26]: Wrong manufacturer.

File: keyboards/handwired/leave_meeting/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 Keebio
```
Comment (tzarc) [inline keyboards/handwired/leave_meeting/keymaps/default/keymap.c:1]: Wrong author.

File: keyboards/handwired/leave_meeting/keymaps/default/config.h (hunk 2 above)
Comment (tzarc) [reply keyboards/handwired/leave_meeting/keymaps/default/config.h:1]: Actually, just delete this file -- it serves no purpose.

File: keyboards/handwired/leave_meeting/keymaps/default/readme.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# The default keymap for tukey
```
Comment (tzarc) [inline keyboards/handwired/leave_meeting/keymaps/default/readme.md:1]: Wrong name. Probably best to just remove the file.

File: keyboards/handwired/leave_meeting/leave_meeting.c [hunk 6]
```diff
@@ -0,0 +1,16 @@
+/* Copyright 2019 Keebio
```
Comment (tzarc) [inline keyboards/handwired/leave_meeting/leave_meeting.c:1]: Wrong author.

File: keyboards/handwired/leave_meeting/leave_meeting.h [hunk 7]
```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 Keebio
```
Comment (tzarc) [inline keyboards/handwired/leave_meeting/leave_meeting.h:1]: Wrong author.

**Review by tzarc — CHANGES_REQUESTED** [review]
Could this not be handled by a keymap under `handwired/onekey` instead?
Seems like it's just a `DIRECT_PINS` definition with `Alt+F4` binding?

File: keyboards/handwired/leave_meeting/rules.mk [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (drashna) [inline keyboards/handwired/leave_meeting/rules.mk:11]:
```suggestion
```

File: keyboards/handwired/leave_meeting/rules.mk [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (drashna) [inline keyboards/handwired/leave_meeting/rules.mk:31]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output 
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**